ccm_web/client/src/pages/MergeSections.tsx

```diff
@@ -209,21 +212,44 @@ function MergeSections (props: CCMComponentProps): JSX.Element {
   const getSelectSections = (): JSX.Element => {
     return (
       <>
-        <Grid container spacing={5} style={{ marginBottom: '0px', marginTop: '0px' }}>
-          <Grid className={classes.sectionSelectionContainer} item xs={12} sm={6}>
-            {getSelectSectionsUnstaged()}
-          </Grid>
-          <Grid className={classes.sectionSelectionContainer} item xs={12} sm={6}>
-            {getSelectSectionsStaged()}
-          </Grid>
+      <Grid container spacing={5} style={{ marginBottom: '0px', marginTop: '0px' }}>
+        <Grid className={classes.sectionSelectionContainer} item xs={12} sm={6}>
+          {getSelectSectionsUnstaged()}
+        </Grid>
+        <Grid className={classes.sectionSelectionContainer} item xs={12} sm={6}>
+          {getSelectSectionsStaged()}
         </Grid>
-        <Button className={classes.submitButton} onClick={submit} variant='contained' color='primary' disabled={!canMerge()}>{mergeButtonText(mergableSections())}</Button>
+      </Grid>
+      <Grid container className={classes.buttonGroup} justifyContent='flex-end'>
+        <Button
+          className={classes.submitButton}
+          onClick={submit}
+          variant='contained'
+          color='primary'
+          disabled={!canMerge()}
+        >
+          {mergeButtonText(mergableSections())}
+        </Button>
+      </Grid>
       </>
     )
   }
 
   const getMergeSuccess = (): JSX.Element => {
-    return (<CourseSectionList canUnmerge={isSubAccountAdminOrAccountAdmin()} {...props}/>)
+    return (
+      <>
+      <CourseSectionList canUnmerge={isSubAccountAdminOrAccountAdmin()} {...props} />
+      <Grid container className={classes.buttonGroup} justifyContent='flex-start'>
+        <Button
+          variant='outlined'
+          aria-label={`Start ${props.title} again`}
+          onClick={() => setPageState(PageState.SelectSections)}
+        >
+          Start Again
+        </Button>
+      </Grid>
```

Comment:
I see that `Start Again` component is used across features. Can we generalize component as `StartAgain` component take props appropriately?

Reply:
We could certainly, but this was what I was trying to say in the meeting: I'm not sure if it confers enough benefits to do so. For example, compare the current usage of out-of-the-box `Button` above to this:

```
<StartAgainButton
  title={props.title} // Used by aria-label
  onClick={() => setPageState(PageState.SelectSections}
/>
```

So the main benefits are we don't have to repeat `variant='outlined'` or `Start Again`, or use a closing tag, and some of the `aria-label` prep is taken care of. That's something. If you want me to do this, I will, I just think it's only providing incremental utility.

We could also include the `Grid` wrapper, but I use this for multiple different kinds of buttons, so I'd be included to have that be it's own `ButtonGroup` component that accepts a flex setting and children.

Reply:
I see your point on this. Let me think about it a bit. I will get back to you tomorrow.

Reply:
Ok, Keep it as you designed now. No need to generalize it